includes/virtual-machines-share-images-across-tenants.md

```diff
@@ -31,6 +31,8 @@ Create an application registration that will be used by both tenants to share th
 1. In **Expires**, leave the default of **In 1 year** and then select **Add**.
 1. Copy the value of the secret and save it to a safe place. You cannot retrieve it after you leave the page.
 
+> [!NOTE]
```

Comment:
Could you possibly include the "results" if these aren't input correctly? The steps already say to do these 2 things, so I'm not sure there is value in repeating those things unless we can give more information. For example, the note could say "If you got an x08 error, make sure you selected xyz"

Reply:
I agree, its redundant, sadly I did not save the errors or take screen shots of the errors.